src/Global/atoms/UserDrawerMeta.tsx

```diff
@@ -75,23 +78,94 @@ const Container = styled.div`
   cursor: pointer;
 `;
 
+const TitleBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const IconBox = styled.div`
+  padding: 1px 5px 0 0;
+`;
+
+const CopyTextBox = styled.pre`
+  padding: 5px;
+  font-size: 10px;
+  line-height: 1.2;
+  box-shadow: 2px 2px 4px (195deg 16% 72% / 72%);
+  border: 1px solid hsl(195deg 5% 60% / 72%);
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  opacity: 0.76;
+  font-weight: 500;
+  color: #575757;
+  position: absolute; 
+  max-width: 170px;
+  white-space: pre-wrap;
+  font-family: ${({ theme }) => theme.fontFamily.data};
+  right:10px;
+  margin-top: -23px;
+`;
+
+const CopyBox = styled.div`
+`;
+
 interface IProps {
   item: IUserDrawerMeta;
   onUserDrawerMetaClick?:() => void;
+  includeCopyTitle?: boolean;
+  copySuccessMessage?: string;
+  userMetaIndex?: number
 }
 
-const UserDrawerMeta : React.FC<IProps> = ({item, onUserDrawerMetaClick}) => {
-  const { icon, title, subTitle, notes } = item;
+interface IShowCopyIcon {
+  id?: number;
+  value?: boolean
+}
+
+
+const UserDrawerMeta : React.FC<IProps> = ({item, onUserDrawerMetaClick, copySuccessMessage, includeCopyTitle, userMetaIndex}) => {
+  const { icon, title, subTitle, notes, hasCopyIcon } = item;
+  const { copyToClipboard } = useCopyToClipboard();
+  const [ showCopyText, setShowCopyText ] = useState<boolean>(false);
+  const [showCopyIcon, setShowCopyIcon] = useState<IShowCopyIcon>({id: 0, value: false});
 
+  const onClickCopyText = useCallback((title , subTitle, notes)=>{
+    let copyText;
+    if(includeCopyTitle){
+      copyText = (title ? title : '') + '\n' + (subTitle ? subTitle : '') + '\n' + (notes ? notes : '');
+    } else {
+      copyText = (subTitle ? subTitle : '') + '\n' + (notes ? notes : '');
+    }
+    copyToClipboard(copyText);
+    setShowCopyText(true);
+    setTimeout(()=>{
+      setShowCopyText(false);
+    }, 300);
```

Comment:
Let's make this a little longer. Let's try `1500`.

Reply:
Done